External/Hide Waiting For Game/Hide Waiting For Game.asm

```diff
@@ -0,0 +1,2 @@
+#To be inserted at 804dec20
+.long 0x00000001
```

Comment:
This file should not be in external. It's part of the playback family of codes. External is for non-slippi stuff